src/syzygy/tbprobe.cpp

```diff
@@ -186,7 +192,13 @@ const char* PawnlessDtzSuffixes[VARIANT_NB] = {
     nullptr,
 #endif
 #ifdef THREECHECK
-    nullptr
+    nullptr,
+#endif
+#ifdef SUICIDE
+    ".stbz",
```

Comment:
Pawnless suffixes and magics are supposed to be crossed over. So this should be `.gtbz`. Likewise for PawnlessWdlSuffixes.